templates/index.html

```diff
@@ -240,7 +240,15 @@
         });
       };
     </script>
-    <script src="https://assets.ubuntu.com/v1/juju-cards-v1.6.0.js"></script>
+    <script src="https://assets.ubuntu.com/v1/juju-cards-v1.7.2.js"></script>
+    <link rel="stylesheet" href="https://assets.ubuntu.com/v1/juju-cards-v1.7.2.css" />
+
+    <script>
+      setTimeout(function() {
```

Comment:
Why is this in a 5s setTimeout?

Is this `updateHead` required? It looks like you've hardcoded the path anyways.

Reply:
Hmmm ... I think I need to hook this function into componentDidMount.